src/components/movie-card/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import './style.less'
+import {Movie} from '../../models/movieDay';
+
+
+interface State {
+}
+
+interface Props {
+    info: Movie;
+ }
+
+
+class MovieCard extends React.Component<Props, State> {
+
+    constructor(props: Props) {
+        super(props);
+
+        this.state = {
+        };
+    }
+
+    formatTime(time: Date) {
+        let date = new Date(time);
+        let movieTime = '';
+        let hours = date.getHours();
+        let minutes = date.getMinutes();
+        if(hours < 10 && minutes < 10) {
+            movieTime = `0${hours}:0${minutes} `;
+        } else if (hours < 10) {
+            movieTime = `0${hours}:${minutes} `;
+        } else if (minutes < 10) {
+            movieTime = `${hours}:0${minutes} `;
+        } else {
+            movieTime = `${hours}:${minutes} `; 
+        }
```

Comment:
Lines 28-36 could be refactored into simpler logic, if 'hours' and 'minutes' are treated like separate blocks:
```suggestion
hours = hours < 10 ? hours : `0${hours}`;
minutes = minutes < 10 ? minutes: `0${minutes}`;
movieTime = `${hours}:${minutes}`
```